t/lib/warnings/signatures

```diff
@@ -0,0 +1,26 @@
+# PREAMBLE use warnings;
+# PREAMBLE use feature 'signatures';
```

Comment:
I thought I'd missed this feature when writing up `t/test_pl.pod`, but it looks like someone snuck it in afterwards :)

I'll update `t/test_pl.pod`.

Reply:
Oh, I only added it a couple of weeks ago :)